test/ob-jq-test.el

```diff
@@ -71,4 +71,16 @@
 (ert-deftest ob-jq-test/simple-execution ()
   "Test simple execution of jq source block."
   (ob-jq-test/test-src-block simple-execution
-    (should (string= "test" (org-babel-execute-src-block)))))
+    (should (string= "\"test\"\n" (org-babel-execute-src-block)))))
```

Comment:
Any idea why this output differs?

Reply:
No, was there a time when this test used to work? All I know is that quotes are expected in the output if the option `-r` is not given. Same goes for the newline which also gets output even without org babel:

    $ jq '.a' <<<'{"a": "test"}'|xxd
    00000000: 2274 6573 7422 0a                        "test".

Reply:
> spaces > tabs

So I suppose this test was not even written by you, because this file is riddled with tabs. I would have untabified it, but I didn't want to make the diff bigger than necessary.

Reply:
I think this is a org-babel thing that might have changed in the meantime. I am pretty sure this worked at some point (4 years ago). I think if you play with the output format for the src-block it will give what the test is testing for. 

The reason for the tabs is probably from a time when I had my emacs-lisp environment misconfigured. 

The changes seem reasonable to me.

Reply:
I went through the documented `:results` options in https://orgmode.org/manual/results.html but couldn't find a combination which would omit the newline. I believe you that the test worked 4 years ago, but I think it must also have been `jq` which behaved differently, because I can't imagine an org-option which would omit the quotes, since they are part of the output.

In any case thanks for merging! Will there be a release which includes this in the foreseeable future?

Reply:
There was both a new jq version and a host of new version of org-babel. I could also have done something with my org config that I wasn't aware of at the time. Thanks for fixing the test though.

The current master should be available in MELPA. For a stable release I usually wait a week or so to see if anyone reports any problems.

Thanks for the contribution! I forgot to say that yesterday.